src/main/java/seedu/address/model/Entity/Mentor.java

```diff
@@ -97,24 +85,19 @@ public void setSubject(SubjectName subject) {
     @Override
     public HashMap<String, String> viewDetailed() {
```

Comment:
Maybe we can just have one getFieldMap() that returns a HashMap of each field and their value in String. I'll just retrieve the corresponding fields when printing for either list or view command.

Reply:
I think it would be better to keep it in model. because lets say if entity's attributes change in the future, we'll only have to change the details in the `viewDetailed` or `viewMinimal` methods once, as opposed to going to every instance where the corresponding fields are used in the controller and changing it, which could cause bugs. wdyt?

Reply:
So when I print for list, I should just go through every key-value pair right? I think for `viewMinimal`, you can just make it a method of `Entity`. I think just getting name and id should be enough. If the user wants more, they can use the `view` command.

Reply:
Got it for the viewMinimal. When you mean `view` command, you're referring to the `viewDetailed` method right?

Reply:
Yup yup